### DISCUSSION TITLE
ALVR without Steam ? (on Linux)

### DISCUSSION THREAD

**Discussion by AndKe** [alvr-org/ALVR#1211] [General]
I bought X-Plane12 
The developers say it should "just work" with VR.
I do have steam - but my X-Plane 12 is not purchased from/via Steam

Can ALVR make X-Plane work anyway?

**Comment by parkerlreed** [comment]
If it uses SteamVR then yes, add it as a non-Steam game and it should be good. 

There may be a launch argument to throw it into a VR compatible mode. 

>Note: Steam VR is a free download that enables X‑Plane 12 to use VR. Steam VR is compatible with the digital download edition, and X‑Plane on Steam.

https://www.x-plane.com/manuals/desktop/

99% chance you'll have to run the Windows version through Proton (Add the game exe as non-Steam game and right click > properties > compatibility)

Then start SteamVR and ALVR and make sure headset is fully connected and showing the Steam VR void/Home (Highly recommend disabling Home in the SteamVR advanced settings). 

Then according to what they say, start X-Plane while all of that is running and go to the VR settings in-game.

**Comment by AndKe** [comment]
@parkerlreed  Thank you... I fail to find any SteamVR for Linux other than this old thing: https://github.com/ValveSoftware/SteamVR-for-Linux

When I run ALVR-x86_64.AppImage  I get "SteamVR not installed: make sure you launched it at least once, then close it"

**Reply by parkerlreed** [reply]
Yeah. In Steam Library do a search for SteamVR. It's just another "game" or app that you can download.

**Comment by AndKe** [comment]
It works!
I can confirm that the nightly .apk + .appimage works fine Ubuntu 22.10
I also managed to run X-Plane from SteamVR - and get it into my headset.
There is one thing that I found odd:
Please see screenshots below. 
The "room" looks .. not that great - but the X-Plane graphics are ..bad. (low resolution)
I see X-Plane rendered on the desktop as well. - is this normal/expected?
I have used the default settings, observed "75% resolution" which is default.
X-Plane is not exactly about high framerate and low latency - which settings do you suggest for really high quality?


![alvr client quest nightly-20221020-190150](https://user-images.githubusercontent.com/1468842/197015374-a5311dc3-f1aa-4fcd-a2f2-386e679eb63a.jpg)

![alvr client quest nightly-20221020-190358](https://user-images.githubusercontent.com/1468842/197015367-deee1826-23bc-42eb-88a6-2f7898dcf05e.jpg)

**Reply by parkerlreed** [reply]
You can up the 75% if you have good hardware. That's a default as to not make a system crap out if it's not as good. 

As mentioned, Home is a hit or miss anyways so I just turn it off within the SteamVR settings. Saves you a tad bit on performance not having to load it and keep it in memory.
